libs/utils/utils.js

```diff
@@ -654,6 +654,16 @@ export function loadPrivacy() {
   });
 }
 
+async function loadJarvisChat() {
```

Comment:
Since this is loaded after delay, this can be moved to it's own file.

Reply:
We would end up loading two files (one with 7 lines of code and the other one with all jarvis chat logic). Do you really think this is necessary? The method is just 7 lines of code and does basic checks whether the jarvis chat logic module should be loaded of not.
The privacy feature is not part of this PR, but we do have some ideas to move it under features, provide the possibility of adding a custom configuration (needed for non *.adobe.com projects) and maybe then we can think about starting a delay file. I remember there have been some problems with this approach in the past in the `blog` project, but can't find the details now - https://github.com/adobe/blog/blob/main/scripts/delayed.js.